src/components/MovieCard/MovieCard.js

```diff
@@ -21,7 +18,10 @@ class MovieCard extends Component {
 
     return (
       <article className="movie-card">
-        <Link to={`/movies/${id}`}>
+        <Link
+          to={`/movies/${id}`}
+          data-testid={`movie-${id}`}
+        >
```

Comment:
I wonder is it common to leave the data-testid in the html itself?  I am not sure if it is common to have these elements in the html.

Reply:
> I wonder is it common to leave the data-testid in the html itself? I am not sure if it is common to have these elements in the html.

Tim, It's technically just adding an html attribute to the element which is totally fine. I had to add that in order to do the integration testing to target the movie card anchor tag within the test to be able to fire the click event.